runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -1323,6 +1323,7 @@ Java_java_lang_invoke_MethodHandleNatives_resolve(
 																	J9UTF8_DATA(signature) + sigOffset,
 																	J9UTF8_DATA(signature) + sigOffset,
 																	J9UTF8_LENGTH(signature) - sigOffset - 1, /* -1 to remove the trailing ;*/
+																	true,
 																	true);
```

Comment:
These arguments should be `TRUE`, not `true` or `false`: the argument types are `BOOLEAN`, not `bool`.